packages/devextreme/js/__internal/ui/radio_group/m_radio_collection.ts

```diff
@@ -59,9 +66,29 @@ class RadioCollection extends CollectionWidget<Properties> {
       const $radioContainer = $('<div>').append($radio).addClass(RADIO_VALUE_CONTAINER_CLASS);
 
       $(itemElement).prepend($radioContainer);
+
+      const aria = {
+        role: 'radio',
+        // eslint-disable-next-line spellcheck/spell-checker
+        labelledby: contentId,
+      };
+
+      this.setAria(aria, $radioContainer);
     }
 
     super._postprocessRenderItem(args);
+
+    if (!html) {
```

Comment:
Looks like it's not mandatory to do it after `super._postprocessRenderItem(args);` => we can move it to the condition above